jobs/config.json

```diff
@@ -11234,72 +11234,6 @@
       "sig-scalability"
     ]
   },
-  "ci-kubernetes-kubemark-5-gce": {
-    "args": [
-      "--cluster=kubemark-5",
-      "--env-file=jobs/env/ci-kubernetes-e2e-kubemark-common.env",
-      "--extract=ci/latest",
-      "--gcp-node-image=gci",
-      "--gcp-node-size=n1-standard-2",
-      "--gcp-nodes=1",
-      "--gcp-project=k8s-jenkins-kubemark",
-      "--gcp-zone=us-central1-f",
-      "--kubemark",
-      "--kubemark-nodes=5",
-      "--provider=gce",
-      "--test=false",
-      "--test_args=--ginkgo.focus=\\[Feature:Empty\\] --gather-resource-usage=true --gather-metrics-at-teardown=true",
-      "--timeout=60m"
-    ],
-    "scenario": "kubernetes_e2e",
-    "sigOwners": [
-      "sig-scalability"
-    ]
-  },
-  "ci-kubernetes-kubemark-5-gce-last-release": {
```

Comment:
I think the reason for keeping this around is to easily find breaking changes on old release (using commit diff). It's a bit hard to find it from presubmits when they start failing. Wdyt?

Reply:
The question is how much do we care for that?

Reply:
Thinking a bit more about it, it feels like keeping it around can be useful (I remember in the past some instances where testinfra changes broke kubemark on older releases, for e.g due to gcloud api incompatibility, bash version, etc). Just to act as a sanity check for kubemark on older release.